src/storage/src/hummock/local_version_manager.rs

```diff
@@ -60,6 +60,16 @@ use crate::hummock::{
 use crate::monitor::StateStoreMetrics;
 use crate::storage_value::StorageValue;
 
+#[derive(Default)]
+pub struct SyncResult {
+    /// The size of all synchronous shared buffers.
+    pub sync_size: usize,
+    /// The sst_info of sync.
+    pub uncommitted_ssts: Vec<LocalSstableInfo>,
+    /// If this epoch is sync, it will be false.
```

Comment:
```suggestion
    /// If this epoch is synced, it will be false.
```

Reply:
Please also document that sync_succeed can be false for a checkpoint barrier because we can sync multiple epochs in one shot.